hbp_knowledge/receptors/zoli2018.bel

```diff
@@ -83,214 +92,337 @@ a(MESH:Acetylcholine) -- bp(MESH:Apoptosis)
+UNSET Confidence
 
 SET Evidence ="Chronic nicotine exposure induces neural adaptations that change cell physi- ology and behaviour mainly as a result of activation and/or desensitisation of nAChRs. Studies of the brains of animals and smokers chronically exposed to nicotine have shown an increase in the number of nAChRs (up-regulation)."
-a(MESH:Nicotine) -| p(MESH:"Receptors, Nicotinic")
+SET Confidence = "Medium"
+a(MESH:Nicotine) -> p(MESH:"Receptors, Nicotinic")
+#in the last line of evidence it is suggested that nicotine upregulates nAChRs. In the line above it is suggested that nicotine leads to activation and/or desensitisation.
+UNSET Confidence
 
 SET Evidence ="The up- regulation of nAChRs has also been obtained using nicotinic agonists (cytisine, carbamylcholine and varenicline) [66, 67], antagonists (dihydro-β-erythroidine, mecamylamine) [68-70] and a partial agonist (CC4) "
-a(HBP:cytisine) -| p(MESH:"Receptors, Nicotinic")
-a(MESH:Carbachol) -| p(MESH:"Receptors, Nicotinic")
-a(MESH:Varenicline) -| p(MESH:"Receptors, Nicotinic")
-a(MESH:"Dihydro-beta-Erythroidine") -| p(MESH:"Receptors, Nicotinic")
-a(MESH:Mecamylamine) -| p(MESH:"Receptors, Nicotinic")
-a(HBP:cytisine) -| act(a(MESH:Nicotine))
-a(MESH:Carbachol) -| act(a(MESH:Nicotine))
-a(MESH:Varenicline) -| act(a(MESH:Nicotine))
+SET Confidence = "Medium"
+a(HBP:cytisine) -> p(MESH:"Receptors, Nicotinic")
+a(MESH:Carbachol) -> p(MESH:"Receptors, Nicotinic")
+a(MESH:Varenicline) -> p(MESH:"Receptors, Nicotinic")
+a(MESH:"Dihydro-beta-Erythroidine") -> p(MESH:"Receptors, Nicotinic")
+a(MESH:Mecamylamine) -> p(MESH:"Receptors, Nicotinic")
+a(HBP:cytisine) -> act(a(MESH:Nicotine))
+a(MESH:Carbachol) -> act(a(MESH:Nicotine))
+a(MESH:Varenicline) -> act(a(MESH:Nicotine))
 a(MESH:"Dihydro-beta-Erythroidine") -| act(a(MESH:Nicotine))
 a(MESH:Mecamylamine) -| act(a(MESH:Nicotine))
-
+#evidence says upregulation of nACHRs in all cases
+UNSET Confidence
 
 SET Evidence ="There is evidence indicating that key steps in nicotine-induced up-regulation are receptor assem- bly [72, 73], decreased proteasomal degradation [74], traf- ficking [75] and cell surface expression "
+SET Confidence = "High"
 a(MESH:Nicotine) -> bp(GO:"nodal receptor complex assembly")
 a(MESH:Nicotine) -> act(p(MESH:"Proteasome Endopeptidase Complex"))
+UNSET Confidence
 
 SET Evidence ="Dopamine (DA) neurons, which project to the NAc receive both excitatory glutamater- gic and cholinergic afferents that mediate nicotine reward, and inhibitory GABAergic afferents, that mediate aversion [77]. The release of these neurotransmitters is modulated by the nAChRs expressed in cholinergic, glutamatergic and GABAergic terminals "
+SET Confidence = "Low"
 a(MESH:"Dopaminergic Neurons") reg a(MESH:Nicotine)
 p(MESH:"Receptors, Nicotinic",loc(MESH:"GABAergic Neurons")) -- a(HBP:neurotransmitters)
 p(MESH:"Receptors, Nicotinic",loc(MESH:"Cholinergic Neurons")) -- a(HBP:neurotransmitters)
```

Comment:
I would set different Neurons as Annotations here ( in Annotation list "Cell" you should finde them all) and then change the statement into regulation of the secretion of neurotransmitter by nachrs
```suggestion
p(MESH:"Receptors, Nicotinic") reg sec(a(HBP:neurotransmitters))
```